components/javax.cache/src/main/java/org/wso2/carbon/caching/impl/Constants.java

```diff
@@ -0,0 +1,7 @@
+package org.wso2.carbon.caching.impl;
+
+public class Constants {
+    public static final int SUPER_TENANT_ID = -1234;
+    public static final String SUPER_TENANT_DOMAIN_NAME = "carbon.super";
+    public static final String TENANT_DOMAIN = "tenantDomain";
```

Comment:
Lets comment  out TENANT_DOMAIN if it is not used anywhere

Reply:
+1 Fixed